settings.gradle.kts

```diff
@@ -59,5 +59,9 @@ if (localProperties["useLocalProjectorClient"] == "true") {
 
 include("projector-agent")
 include("projector-awt")
+include("projector-awt-common")
+include("projector-awt-jdk17")
 include("projector-plugin")
 include("projector-server")
+include("projector-server-common")
+include("projector-server-jdk17")
```

Comment:
In modules that require jdk17 toolchain is specified explicitly so even you started building with 11 gradle will pick 17 (download if needed) and use it to build those modules. I missed the vice-versa situation, fixing it new commit

Reply:
But IDEA currently doesn't pick up java version from gradle so will show errors in editor if code and module java version mismatch. I can add condition, but then there should be separate CI workflows to test so that test from all modules are checked. So I think it's better to leave it unconditional